### PR TITLE
fix: register problems

### LINKED ISSUE #357: email field not detected as email
It seems the email field is not detected as a real email field by the browser (no suggestion, no firefox relay, etc). Make sure the correct html input type is used.

### LINKED ISSUE #360: Sales tax is not collected on any wallet atm
Sales tax is not collected on a different wallet on front-end

### LINKED ISSUE #362: option to live in the USA is not correctly aligned
image

should be horizontal

### PR DESCRIPTION
**Pull request by fricoben**
close #362 
close #360
close #357

### CODE REVIEW

File: utils/feltService.ts [hunk 1]
```diff
@@ -37,6 +37,25 @@ export function gweiToEth(gwei: string): string {
   return ethBigInt.toString();
 }
 
+export function multiplyBigIntByPercentage(
```
Comment (Th0rgal) [inline utils/feltService.ts]: rename as applyRateToBigInt because you pass a rate (0-1) and not a percentage

**Review by Th0rgal — CHANGES_REQUESTED** [review]
looks good to me, please juste update the percentage function

**Review by irisdv — APPROVED** [review]
lgtm, it looks way better with the calls refactored !

**Review by Th0rgal — APPROVED** [review]
lgtm

**Pull request merged**